deploy/upstream/manifests/0.17.0/0000_50_olm_15-packageserver.clusterserviceversion.yaml

```diff
@@ -70,7 +70,7 @@ spec:
         spec:
           strategy:
             type: RollingUpdate
-          replicas: 2
+          replicas: 1
```

Comment:
+1, the same goes for the change to `deploy/ocp/manifests/0.17.0/0000_50_olm_15-packageserver.clusterserviceversion.yaml`.

Reply:
Ok, I've reverted the changes on the released manifests. The quickstart manifest is still using single replica packageserver.

Reply:
Wait, hold on, I think we do need this change to land in the downstream `manifests/` directory (unless we want to only make that change in the downstream repo?), we just don't want it to land in any of the `/0.17.0` release directories because those releases are already, well, released.

Reply:
So we want to update replicas in the ocp manifests then?